lib/src/Uncertainty/Distribution/Normal.cxx

```diff
@@ -653,7 +653,7 @@ Point Normal::computeSequentialConditionalQuantile(const Point & q) const
       result[i] = mean_[i] + sigma_[i] * DistFunc::qNormal(q[i]);
     return result;
   }
-  return mean_ + inverseCholesky_.solveLinearSystem(DistFunc::qNormal(q));
+  return EllipticalDistribution::computeSequentialConditionalQuantile(q);
```

Comment:
I don't understand what has motivated this change. Using the following script:
```python
import openturns as ot

# Build a multivariate Normal distribution
R = ot.CorrelationMatrix(3, [1.0, 0.5, 0.2, 0.5, 1.0, 0.3, 0.2, 0.3, 1.0])
mu = [1.0, 2.0, 3.0]
sigma = [2.0, 1.5, 3.0]
distribution = ot.Normal(mu, sigma, R)
# Extract its iso-proba tranformation and its inverse
T = distribution.getIsoProbabilisticTransformation()
invT = distribution.getInverseIsoProbabilisticTransformation()
# Build the corresponding Rosenblatt transformations
Ros = ot.Function(ot.RosenblattEvaluation(distribution))
invRos = ot.Function(ot.InverseRosenblattEvaluation(distribution))
# Check if the composition is the identity
# First, the Nataf version
sample = distribution.getSample(100)
res = invT(T(sample))
print("err Nataf=", (res - sample).computeRawMoment(2))
# Second, the Rosenblatt version
res = invRos(Ros(sample))
print("err Ros=", (res - sample).computeRawMoment(2))
# Third, the Rosenblatt/Nataf mix
res = invRos(T(sample))
print("err Ros/Nat=", (res - sample).computeRawMoment(2))
# Fourth, the Nataf/Rosenblatt mix
res = invT(Ros(sample))
print("err Nat/Ros=", (res - sample).computeRawMoment(2))
```
I gat with the current master:
```
err Nataf= [0,2.66241e-32,1.72563e-32]
err Ros= [2.55036e-30,8.69621e-30,3.15051e-30]
err Ros/Nat= [2.5103e-30,8.67994e-30,3.04993e-30]
err Nat/Ros= [2.5103e-30,8.67402e-30,3.1091e-30]
```
as expected, and with your branch:
```
err Nataf= [0,2.66241e-32,1.72563e-32]
err Ros= [2.55036e-30,0.00983886,0.125701]
err Ros/Nat= [2.5103e-30,0.00983886,0.125701]
err Nat/Ros= [2.5103e-30,8.67402e-30,3.1091e-30]
```
which shows a problem...